vmdb/app/models/mixins/password_mixin.rb

```diff
@@ -13,6 +18,9 @@ def encrypt_column(column)
       #   password_encrypted  : Get the password in cryptext
       #   password_encrypted= : Set the password in cryptext
 
+      encrypted_columns << column.to_s
+      encrypted_columns << "#{column}_encrypted"
```

Comment:
not sure that `column_encrypted` is an `encrypted_columns`. Thought only `column.to_s` would be in there